packages/@webex/plugin-meetings/src/reachability/index.ts

```diff
@@ -915,116 +950,84 @@ export default class Reachability extends EventsScope {
 
       // initialize the result for this cluster
       results[key] = {
-        udp: {result: cluster.udp.length > 0 ? 'unreachable' : 'untested'},
-        tcp: {result: cluster.tcp.length > 0 ? 'unreachable' : 'untested'},
-        xtls: {result: cluster.xtls.length > 0 ? 'unreachable' : 'untested'},
+        udp: {result: cluster.udp.length > 0 ? 'unreachable' : 'untested', details: []},
+        tcp: {result: cluster.tcp.length > 0 ? 'unreachable' : 'untested', details: []},
+        xtls: {result: cluster.xtls.length > 0 ? 'unreachable' : 'untested', details: []},
         isVideoMesh: cluster.isVideoMesh,
       };
 
-      // update expected results counters to include this cluster
-      this.expectedResultsCount[cluster.isVideoMesh ? 'videoMesh' : 'public'].udp +=
-        cluster.udp.length > 0 ? 1 : 0;
+      if (cluster.udp.length) {
+        this.expectedResultsCount[cluster.isVideoMesh ? 'videoMesh' : 'public'].udp += 1;
+      }
       if (!cluster.isVideoMesh) {
-        this.expectedResultsCount.public.tcp += cluster.tcp.length > 0 ? 1 : 0;
-        this.expectedResultsCount.public.xtls += cluster.xtls.length > 0 ? 1 : 0;
+        if (cluster.tcp.length) this.expectedResultsCount.public.tcp += 1;
+        if (cluster.xtls.length) this.expectedResultsCount.public.xtls += 1;
       }
     });
 
-    const isFirstResult = {
-      udp: true,
-      tcp: true,
-      xtls: true,
-    };
-
-    // save the initialized results (in case we don't get any "resultReady" events at all)
     await this.storeResults(results);
-
-    if (!clusterList || !Object.keys(clusterList).length) {
-      // nothing to do, finish immediately
-      this.resolveReachabilityPromise(false);
-
-      this.emit(
-        {
-          file: 'reachability',
-          function: 'performReachabilityChecks',
-        },
-        'reachability:done',
-        {}
-      );
-
-      return;
-    }
-
     this.startTimers();
 
-    // now start the reachability on all the clusters
     Object.keys(clusterList).forEach((key) => {
       const cluster = clusterList[key];
 
-      this.clusterReachability[key] = new ClusterReachability(key, cluster);
-      this.clusterReachability[key].on(Events.resultReady, async (data: ResultEventData) => {
-        const {protocol, result, clientMediaIPs, latencyInMilliseconds} = data;
-
-        if (isFirstResult[protocol]) {
-          this.emit(
-            {
-              file: 'reachability',
-              function: 'resultReady event handler',
-            },
-            'reachability:firstResultAvailable',
```

Comment:
what happened to this code? why is it deleted? why are we not sending "reachability:firstResultAvailable" event anymore?